pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -123,9 +130,18 @@ func newCertRotationController(
 		klog.Warningf("!!! UNSUPPORTED VALUE SET !!!")
 		klog.Warningf("Certificate rotation base set to %q", rotationDay)
 	} else {
-		// for the development cycle, make the rotation 60 times faster (every twelve hours or so).
-		// This must be reverted before we ship
-		rotationDay = rotationDay / 60
+		// For the internal CI testing, make the cert rotation much faster.
+		// This is RH infrastructure specific, dirty, but it gets the job done without need to revert/apply this on every dev cycle start
+		if !cache.WaitForCacheSync(ctx.Done(), configInformer.Config().V1().Infrastructures().Informer().HasSynced) {
+			utilruntime.HandleError(fmt.Errorf("caches did not sync"))
+			return nil, errors.New("failed to wait for infrastructure informer")
+		}
+		if infra, err := ret.infrastructureLister.Get("cluster"); err == nil {
+			if strings.HasPrefix(infra.Status.InfrastructureName, "ci-op") && strings.Contains(infra.Status.APIServerURL, "dev.rhcloud.com") {
```

Comment:
@mfojtik Please also add one more condition 'devcluster.openshift.com' of the cluster APIServer URL for QE testing. We have been using this rotationDay setting in our DR Cert testing.